### PR TITLE
a color picker for GUI

### PR DESCRIPTION
**Pull request by abow**

### CODE REVIEW

File: gui/src/controls/colorpicker.ts [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            this._updateValueFromPointer(coordinates.x, coordinates.y);
+            this._host._capturingControl = this;
+
+            super._onPointerDown(coordinates);
+        }
+
+        protected _onPointerMove(coordinates: Vector2): void {
+            if (this._pointerIsDown) {
+                this._updateValueFromPointer(coordinates.x, coordinates.y);
+            }
```
Comment (deltakosh) [inline gui/src/controls/colorpicker.ts]: need to call super._onPointerMove

**Comment by deltakosh** [comment]
This is really cool
Need to update the doc as well :)

**Comment by deltakosh** [comment]
I'll do the quick fix :)

can you take care of the doc?

**Comment by abow** [comment]
Yes, I'll do it tonight.  Thanks

**Comment by deltakosh** [comment]
Thanks a lot!

**Pull request merged**